modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/Rails5ServerCodegen.java

```diff
@@ -281,8 +281,8 @@ public String toApiFilename(String name) {
         // replace - with _ e.g. created-at => created_at
         name = name.replaceAll("-", "_"); // FIXME: a parameter should not be assigned. Also declare the methods parameters as 'final'.
 
-        // e.g. PhoneNumberApi.rb => phone_number_api.rb
-        return underscore(name) + "_controllers";
+        // e.g. DefaultController => defaults_controller.rb
+        return underscore(name) + "s_controller";
```

Comment:
@bluengreen thanks for the PR, will review and let you know if we've any question.